### DISCUSSION TITLE
the path /shfj cannot be found on your system

### DISCUSSION THREAD

**Discussion by DimitriPapadopoulos** [brainvisa/brainvisa.github.io#62] [Q&A] [answered]
The local version of Anatomist emits the following warning:
```
WARNING: the path /shfj cannot be found on your system, so it cannot be mounted in the container as requested by your casa-distro configuration.
```
Is there a way to get rid of it?

**Comment by DimitriPapadopoulos** [comment]
The above error messages adds clutter, especially associated to other error messages, for example when Anatomist does not start for lack of a proper version of Singularity:
```
WARNING: the path /shfj cannot be found on your system, so it cannot be mounted in the container as requested by your casa-distro configuration.
ERROR  : Unknown image format/type: /i2bm/brainvisa/brainvisa-cea-5.0.2.sif
ABORT  : Retval = 255
Your version of Singularity (2.6.1-dist) is not supported, please install Singularity 3.0.0 or later (see https://brainvisa.info/).
```

**Comment by ylep** [comment, marked as answer]
Hi @DimitriPapadopoulos, the reason for this message is that `/shfj` is included in your casa-distro configuration. To remove the warning, you can launch the bare `bv` command, which will launch a GUI where you can remove the `/shfj` mount from the configuration. Alternatively, you can manually remove the `"/shfj": "/shfj"` line from `~/.config/casa-distro/casa_distro_3.json`.

The reason `/shfj` got into that file is that this configuration file is automatically created the first time you use brainvisa (or any containerized command through casa-distro). The standard paths (`/home`, `/mnt`, etc.) as well as our CEA-specific paths (`/neurospin`, `/shfj`...) are added to the config if they are detected at that time.

When you come back later on another machine that does not have `/shfj`, casa-distro will complain that it is missing. I agree that the message is confusing and mostly useless... Maybe we should not add the CEA-specific paths to that configuration file, and perform the auto-detection every time?

**Comment by DimitriPapadopoulos** [comment]
`/shfj` is not available on CEA networks anymore.

**Reply by ylep** [reply]
@DimitriPapadopoulos thanks for the info, I have removed it from the list of “standard” paths to automatically configure (https://github.com/brainvisa/casa-distro/commit/965982b25c1ca789f868c7ef054bc6b59dafe7e0).
